src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -42,7 +42,17 @@ repos:
         - file
         - python
 {% endif %}
-{%- if cookiecutter.use_flake8 == "yes" %}
+
+{%- elif cookiecutter.auto_format_tool == 'ruff' +%}
```

Comment:
Also is the `+` at the end intentional?

Reply:
+1 for what @Saransh-cpp mentioned!
@ayeankit please keep the same standard used by the other options, keep the same standard for jinja tags and keep empty lines just when the other ones has empty lines, otherwise don't keep extra lines when not necessary. let's try to keep the entire file in the same visual style as possible.